### PR TITLE
add org.gtk.Gtk3theme.High-Sierra flatpak theme

### PR DESCRIPTION
**Pull request by chengjk84**
Here by I attach manifest file for High Sierra theme which compressed version maintain by me at https://github.com/chengjk84/high-sierra-gtk-theme

And the source version maintain by Vince at https://github.com/vinceliuice/Sierra-gtk-theme

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build org.git.Gtk3theme.High-Sierra

**Comment by nedrichards** [comment]
bot, build org.gtk.Gtk3theme.High-Sierra

**Comment by chengjk84** [comment]
done building it...

**Comment by bilelmoussaoui** [comment]
bot, build doesn't mean that you have to build the package and push the changes. It's used by the builder bot to trigger a new test build. You need to revert your latest commit

File: org.gtk.Gtk3theme.High-Sierra.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+    <id>org.gtk.Gtk3theme.High-Sierra</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <name>High Sierra GTK Theme</name>
+    <summary>High Sierra like theme for GTK 2 and GTK 3.</summary>
```
Comment (TingPing) [inline org.gtk.Gtk3theme.High-Sierra.appdata.xml]: Might as well just remove the GTK 2 reference since this is a GTK3 only package.

Comment (chengjk84) [reply org.gtk.Gtk3theme.High-Sierra.appdata.xml]: done remove it...

**Comment by chengjk84** [comment]
sorry... first timer... everything is updated to like what is before with some description update...

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gtk.Gtk3theme.High-Sierra

**Pull request closed without merging**